mastering-plone/testing.rst

```diff
@@ -165,16 +165,16 @@ First, we add a simple test for the custom template in our Functional Test layer
 Exercise 1
 ^^^^^^^^^^
 
-We already wrote a talklistview and it is untested!
+We already wrote a Talklistview and it is untested!
```

Comment:
Should this be ``talklistview``, assuming it is inline code?

Reply:
Might be, but I was fixing the capitalization to make it match the other mention, though I did not check the code to see which variant was correct. Consistency over correctness!

Reply:
https://github.com/plone/training/search?q=Talklistview&unscoped_q=Talklistview

It depends on whether you intend the class ``TalkListView`` or the URL or template ``talklistview``. Either way, it should be surrounded by double backticks because they are all code or code-ish enough.

Reply:
`TalkListView` would be correct.